### PR TITLE
[TFLite 16x8, documentation] Added section on 16x8 quantization scheme to model-optimization.

### PR DESCRIPTION
**Pull request by wwwind**
In this PR the section on the 16x8 quantization scheme is added to model-optimization overview document.

### CODE REVIEW

File: tensorflow/lite/g3doc/performance/model_optimization.md [hunk 1]
```diff
@@ -125,6 +125,37 @@ the numbers here:
   </figcaption>
 </figure>
 
+### Full integer quantization with int16 activations and int8 weights
+
+[Quantization with int16 activations](https://www.tensorflow.org/model_optimization/guide/quantization/post_training) is a full integer quantization scheme with activations in int16 and weights in int8. This mode can improve accuracy of the quantized model in comparison to the full integer quantization scheme with both activations and weights in int8 keeping a similar model size. It is recommended when activations are sensitive to the quantization.
```
Comment (jianlijianli) [inline tensorflow/lite/g3doc/performance/model_optimization.md:130]: Could you please mention the expected performance comparison between the 16x8 quantization and the 8x8 quantization? This should give the reader more information for them to make the decision. Thanks.

**Review by jianlijianli — DISMISSED** [review]

File: tensorflow/lite/g3doc/performance/model_optimization.md (hunk 1 above)
Comment (wwwind) [reply tensorflow/lite/g3doc/performance/model_optimization.md:130]: Hi @jianlijianli I added explanation regarding performance. Could you please take a look and re-approve if it is ok? Thanks

**Review by jianlijianli — DISMISSED** [review]
Thanks!

File: tensorflow/lite/g3doc/performance/model_optimization.md [hunk 2]
```diff
@@ -125,6 +125,39 @@ the numbers here:
...
+      <td>7.2%</td></tr>
+    <tr><td>DeepSpeech 0.51 (unrolled)</td><td>CER</td><td>6.13%</td><td>43.67%</td>
+      <td>6.52%</td></tr>
+    <tr><td>YoloV3</td><td>mAP(IOU=0.5)</td><td>0.577</td><td>0.563</td>
+      <td>0.574</td></tr>
+    <tr><td>MobileNetV1</td><td>Top-1 Accuracy</td><td>0.7062</td><td>0.694</td>
+      <td>0.6936</td></tr>
+    <tr><td>MobileNetV2</td><td>Top-1 Accuracy</td><td>0.718</td><td>0.7126</td>
+      <td>0.7137</td></tr>
+    <tr><td>MobileBert</td><td>F1(Exact match)</td><td>88.81(81.23)</td><td>2.08(0)</td>
```
Comment (jdduke) [inline tensorflow/lite/g3doc/performance/model_optimization.md:153]: Which MobileBERT model are you testing? @renjie-liu as FYI

File: tensorflow/lite/g3doc/performance/model_optimization.md [hunk 3]
```diff
@@ -125,6 +125,39 @@ the numbers here:
   </figcaption>
 </figure>
 
+### Full integer quantization with int16 activations and int8 weights
+
+[Quantization with int16 activations](https://www.tensorflow.org/model_optimization/guide/quantization/post_training) is a full integer quantization scheme with activations in int16 and weights in int8. This mode can improve accuracy of the quantized model in comparison to the full integer quantization scheme with both activations and weights in int8 keeping a similar model size. It is recommended when activations are sensitive to the quantization.
+
+Currently only non-optimized reference kernel implementation is available in TFLite so that by default the performance will be slow compared to int8 kernels. Full advantages of this mode can currently be accessed via specialised hardware, or custom software.
```
Comment (jdduke) [inline tensorflow/lite/g3doc/performance/model_optimization.md:132]: This should be emphasized in some fashion, e.g.,

*NOTE*: Currently only non-optimized...

Comment (jdduke) [inline tensorflow/lite/g3doc/performance/model_optimization.md:132]: kernel implementations are available in TFLite for this quantization scheme, so by default the performance...

**Review by jdduke — CHANGES_REQUESTED** [review]

File: tensorflow/lite/g3doc/performance/model_optimization.md (hunk 2 above)
Comment (wwwind) [reply tensorflow/lite/g3doc/performance/model_optimization.md:153]: identical to this one: https://github.com/google-research/google-research/tree/master/mobilebert

**Comment by wwwind** [comment]
Hi @jdduke Corrected, could you please re-approve ? thanks.

File: tensorflow/lite/g3doc/performance/model_optimization.md [hunk 4]
```diff
@@ -125,6 +125,39 @@ the numbers here:
...
+  <table>
+    <tr>
+      <th>Model</th>
+      <th>Accuracy metric type </th>
+      <th>Accuracy (float32 activations) </th>
+      <th>Accuracy (int8 activations) </th>
+      <th>Accuracy (int16 activations) </th>
+    </tr> <tr><td>Wav2letter</td><td>WER</td><td>6.7%</td><td>7.7%</td>
+      <td>7.2%</td></tr>
+    <tr><td>DeepSpeech 0.51 (unrolled)</td><td>CER</td><td>6.13%</td><td>43.67%</td>
```
Comment (akarmi) [inline tensorflow/lite/g3doc/performance/model_optimization.md:145]: nit: 0.51 -> 0.5.1

**Review by akarmi — CHANGES_REQUESTED** [review]

**Review by akarmi — APPROVED** [review]
Thanks!

**Review by multiverse-tf — APPROVED** [review]

**Pull request merged**